onadata/apps/logger/models/xform_version.py

```diff
@@ -0,0 +1,30 @@
+"""
+Module containing the XForm Version model
+"""
+from django.db import models
+
+
+class XFormVersion(models.Model):
+    """
+    XForm version model
+
+    The main purpose of this model is to store the various
+    versions of an XForm & link them to their respective xls files on the
+    storage backend for utilization in the future when a user requires
+    the previous XForm versions XML or JSON.
+    """
+    xform = models.ForeignKey(
+            'logger.XForm', on_delete=models.CASCADE, related_name='versions')
+    # TODO: Test to see if the file is re-uploaded or renamed during the
+    # creation process. The main idea is just to link to the existing file
+    # and not recreate it
+    xls = models.FileField()
+    version = models.CharField(max_length=100)
+    date_created = models.DateTimeField(auto_now_add=True)
```

Comment:
Could we ask who created/uploaded the version/XLSForm? Have a `date_modified` field?